jsHelper/spicetifyWrapper.js

```diff
@@ -1842,7 +1842,9 @@ Spicetify.Playbar = (function () {
 		constructor(label, icon, onClick = () => {}, disabled = false, active = false, registerOnCreate = true) {
 			this.element = document.createElement("button");
 			this.element.classList.add("main-genericButton-button");
-			this.element.style.display = "block";
+			this.iconElement = document.createElement("span");
+			this.iconElement.classList.add("IconWrapper__Wrapper-sc-16usrgb-0", "Wrapper-sm-only");
```

Comment:
I don't see this as being a valuable addition as we'd be using Spotify's weird classnames, which we already try to run off of using the css mapper. 🤔

Reply:
i literally explained the reason in the description.

Also we do this constantly and it's unchanged since we have a css map.

Reply:
maybe it's achievable using `Spicetify.ReactComponent.IconComponent` like we do with the navbar icon links.